src/dev/jest/config.js

```diff
@@ -53,7 +53,7 @@ export default {
     '!src/legacy/core_plugins/**/__snapshots__/**/*',
   ],
   moduleNameMapper: {
-    '^plugins/([^\/.]*)/(.*)': '<rootDir>/src/legacy/core_plugins/$1/public/$2',
+    '^plugins/([^\/.]*)(.*)': '<rootDir>/src/legacy/core_plugins/$1/public$2',
```

Comment:
context is to allow nested folders?

Reply:
rather the opposite, to allow importing from the plugin directly, like `plugin/data`, but to be honest that's not needed anymore so I can actually revert that change here.

Reply:
Well but since we might hit that problem another time I might just leave it, since you anyway already approved it :)